integration-tests/docling/src/test/java/org/apache/camel/quarkus/component/docling/it/DoclingTestResource.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.docling.it;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class DoclingTestResource implements QuarkusTestResourceLifecycleManager {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DoclingTestResource.class);
+    private static final String DOCLING_IMAGE = "quay.io/docling-project/docling-serve:latest";
```

Comment:
Please define the container image name as a Maven property in the root pom.xml under [here](https://github.com/apache/camel-quarkus/blob/main/pom.xml#L229C19-L229C29). Then you can use `ConfigProvider` to look it up. See examples in other `TestResource` classes.

Also, please pin to a specific tag if possible instead of `latest`.